client/signup/jetpack-connect/style.scss

```diff
@@ -90,3 +90,15 @@
 		width: 100%;
 	}
 }
+
+
+.jetpack-connect__redirecting {
+	width: 100%;
+	z-index: 3;
```

Comment:
oops...I was having problems with the z-index() function down below and I added this one to test that it was doing anything ... it shouldn't be there anymore :)